spec/person.rb

```diff
@@ -1,13 +1,20 @@
-class Person
+require './nameable'
+
+class Person < Nameable
   attr_accessor :id, :name, :age
 
   def initialize(age, name: 'Unknown', parent_permission: true)
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your `name: 'Unknown'` something like `name = 'Unknown'` because whenever you give a string as a default value then you always use the equal sign `(=)` 👍 but when you give a boolean as a default value then you use colon `(:)` according to the rules of ruby 😊